gaphor/diagram/shapes.py

```diff
@@ -57,6 +64,7 @@ def __init__(self, *children, style={}, draw=None):
         self._draw_border = draw
 
     def size(self, cr):
+        global Padding
```

Comment:
If you inherit your Padding class from `Enum`, would this avoid declaring this a global?